### DISCUSSION TITLE
How to make a backup?

### DISCUSSION THREAD

**Discussion by igor17721** [FreshRSS/FreshRSS#5879] [Q&A]
I would like to export all of my current settings, including added categories and feeds. Can you help me, please?

**Comment by Alkarex** [comment]
Check our documentation:
* http://freshrss.github.io/FreshRSS/en/admins/05_Backup.html
* https://freshrss.github.io/FreshRSS/en/users/04_Subscriptions.html#import--export

A few notes:
* To export your feeds and categories, the more portable approach is to export as OPML using the Web interface.
* To export all your articles (including feeds and categories), the best is to [export as SQLite using our command line](https://github.com/FreshRSS/FreshRSS/blob/edge/cli/README.md)
* In any case, your settings should be backed-up from `./FreshRSS/data/config.php` and `./FreshRSS/data/users/igor/config.php`

**Reply by Kenqr** [reply]
How do I backup if I installed FreshRSS with docker?

**Reply by Alkarex** [reply]
@Kenqr It depends on what you want to back-up, what kind of database you have, how your Docker volumes are set up, and whether you want to back-up a single user or all users.

With Docker, you can in any case back-up your Docker volumes using Docker commands.

You can also use [FreshRSS command line](https://github.com/FreshRSS/FreshRSS/blob/edge/cli/README.md) by just prefixing them with `docker exec --user www-data freshrss cli/...` such as:

```sh
docker exec --user www-data freshrss cli/export-sqlite-for-user.php --user alice --filename data/users/alice/alex-20240614.sqlite
docker cp "freshrss:/var/www/FreshRSS/data/users/alice/alice-20240614.sqlite" "/home/you/db-alice_20240614.sqlite"
```

See also:
* https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#migrate-database
* http://freshrss.github.io/FreshRSS/en/admins/05_Backup.html
